### DISCUSSION TITLE
Question about wave forecast from api

### DISCUSSION THREAD

**Discussion by markdrecoll** [weather-gov/api#648] [General]
Hi there, I was hoping to use your api for wave forecasts in Chicago.
https://marine.weather.gov/MapClick.php?zoneid=lmz741
This is your web page for the marine weather and it gives the next few days, and a wave range like 2-4ft, subsiding to 1-2 feet. This is perfect and I wondered if there was a way to hit this information through the API.

I have gone through the API docs and no matter how much I look none of the options seems to show this information. The only api call I found that even seemed to list waveHeight is  https://api.weather.gov/gridpoints/LOT/76,75 But that has weird hours, and displays just things like 0.6096m which is 2 feet. But an exact wave number is not as accurate as when the marine weather forecast has like 3-5 feet. Also sometimes it just displays 0. And the waves would never just be 0 so those do not help. And it also has seemingly arbitrary times/dates like listing midnight, then 3 am, then jumping ahead 4 days at the end.

I was wondering if there is any way to just do an api call that receives the marine zone forecast for zone LMZ741.

**Comment by dwhitemv25** [comment]
If you want the text like from marine.weather.gov, you'll have to write your own code for that. Post #427 details why.

What you're seeing in the API is the raw data that goes into producing the MapClick forecast. If you follow the "Tabular Data" link in the footer of your MapClick page (which goes to https://marine.weather.gov/MapClick.php?lat=41.9777&lon=-87.5834&unit=0&lg=english&FcstType=digital), you will see the same individual data points that the API is providing. You'll see that the text forecast is reciting the minimum and maximum heights forecast over the time period.

Read https://weather-gov.github.io/api/gridpoints for an explanation of how the time periods work. 

Judicious use of rounding can help with unit conversion issues.
